### PR TITLE
[Bookaroom 17] - Fixes

### PR DESCRIPTION
**Pull request by Dairai**
- Correct date time formatter
- updating returned property names

### CODE REVIEW

File: src/com/comeet/Room.java [hunk 1]
```diff
@@ -161,7 +161,7 @@ public void addFreeBusyTime(FreeBusySlot slot) {
     }
 
     @XmlElement(name = FREEBUSY)
-    public List<FreeBusySlot> getFreeBusy() {
```
Comment (kb000) [inline src/com/comeet/Room.java:164]: I don't think you need to change the class member names.  The XmlElement name parameters should have fixed the generated json.

File: src/com/comeet/RoomsDao.java [hunk 2]
```diff
@@ -313,13 +313,14 @@ public void populateMetadata(Room room) {
         DateTime startTime = null;
         DateTime endTime = null;
 
-        DateTimeFormatter fmt = ISODateTimeFormat.dateTime();
+        DateTimeFormatter fmt = ISODateTimeFormat.dateTimeNoMillis();
```
Comment (kb000) [inline src/com/comeet/RoomsDao.java:316]: 👍

File: src/com/comeet/Room.java (hunk 1 above)
Comment (kb000) [reply src/com/comeet/Room.java:164]: If it works, it works. Please add a unit test to prove it.

Comment (Dairai) [reply src/com/comeet/Room.java:164]: That's what I thought too but I went through it with Ricardo it still had the incorrect property values

**Review by kb000 — APPROVED** [review]
Suggested some changes, but feel free to merge in order to unblock the app.

**Pull request merged**